.thoth.yaml

```diff
@@ -9,3 +9,11 @@ runtime_environments:
       version: "7"
     python_version: "3.6"
     recommendation_type: latest
+
+managers:
+  - name: thoth-advise
+    configuration:
+      labels: [bot, kebechet]
+  - name: thoth-provenance
+    configuration:
+      labels: [bot, kebechet
```

Comment:
`labels: [bot, kebechet]`, :smile:

Reply:
Just making sure if you actually review pull requests :angel: :see_no_evil: